public/regdesk.html

```diff
@@ -240,9 +240,9 @@ <h5 class="modal-title">Printer Settings For: <span id="printerSettingsSerial"><
                 </div>
               </div>
               <div class="col-md-4">
-                <label for="printerSettingsDarkness" class="col-sm-6 col-form-label">Darkness</label>
+                <label for="printerSettingsDarkness" class="form-label">Print Darkness</label>
                 <select id="printerSettingsDarkness" class="form-select" aria-label="Darkness">
```

Comment:
Are these settings retrievable in some form? So that someone can read how a printer is currently configured when they pull up this screen?

Reply:
This interface would be the way to find them, yes.